src/styles/ingredients.scss

```diff
@@ -0,0 +1,182 @@
+.ingredients {
+  margin-bottom: 100px;
+
+  @include tablet {
+    position: relative;
+    margin-bottom: 130px;
+  }
+
+  @include desktop {
+    max-width: 1280px;
+    margin-bottom: 201px;
+  }
+
+  @include big-desktop {
+    max-width: 1440px;
+  }
+
+  &__title {
+    position: relative;
+    max-width: 280px;
+    margin-bottom: 20px;
+
+    @include tablet {
+      margin-bottom: 310px;
+    }
+
+    @include desktop {
+      max-width: 380px;
+      margin-bottom: 287px;
+    }
+  }
+
+  &__blok {
+    display: block;
+    background: url(../images/svg/bank-mobile.svg) no-repeat right/contain;
+    width: 100%;
+    height: 397px;
+    position: relative;
+    margin-bottom: 30px;
+    right: -20px;
+
+    @include tablet {
+      position: absolute;
+      background: url(../images/svg/bank-tablet.svg) no-repeat right/contain;
+      bottom: 35%;
+      right: 0;
+    }
+
+    @include desktop {
+      position: absolute;
+      width: 600px;
+      background: url(../images/svg/bank.svg) no-repeat right/contain;
+      bottom: 35%;
+      right: 50px;
+    }
+
+    @include big-desktop {
+      right: 120px;
+    }
+  }
+
+  &__list {
+    display: flex;
+    flex-direction: column;
+    gap: 52px;
+    text-align: right;
+    white-space: nowrap;
+    position: absolute;
+    padding: 33px 190px 0 0;
+    right: 10px;
```

Comment:
The 'padding' values seem to be quite large. This might lead to issues with the layout on smaller screens. Please review these values.